pnpm-lock.yaml

```diff
@@ -3782,6 +3789,11 @@ packages:
       is-buffer: 1.1.6
     dev: true
 
+  /meaning-of-life/1.0.0_o3deharooos255qt5xdujc3cuq:
+    resolution: {integrity: sha512-fVA4xSydqtK9owabGcYw1r4EKEsMOVVeYQLeCXPu77Z+8Y2j2B2I16UqZlKIOHnYkJ4RSvpJ00ywy9IWjmuxYw==}
+    dev: true
+    patched: true
+
```

Comment:
Package entries in `pnpm-lock.yaml` can have the `patched` arg, and not how the alias changes, similar to peer deps.